### ISSUE TITLE
model should generate tensorboard data

### ISSUE THREAD

**Issue by Natanande** [joniz7/rnn-chatbot#13]

**Comment by Natanande** [comment]
Should at least save training loss, validation loss and perhaps learning rate. (also perplexity?)

**Comment by Natanande** [comment]
1ac4eeffdb5e49ec62e426139876fa240fe9230e
